src/org/thoughtcrime/securesms/mms/MediaConstraints.java

```diff
@@ -28,6 +29,16 @@
 public abstract class MediaConstraints {
   private static final String TAG = MediaConstraints.class.getSimpleName();
 
+  private static final int LOW_VIDEO_COMPRESSION_HEIGHT  = 1080;
+  private static final int LOW_VIDEO_COMPRESSION_WIDTH  = 1920;
+  private static final int LOW_VIDEO_COMPRESSION_BITRATE = 750000;
+  private static final int MEDIUM_VIDEO_COMPRESSION_HEIGHT  = 720;
+  private static final int MEDIUM_VIDEO_COMPRESSION_WIDTH  = 1280;
+  private static final int MEDIUM_VIDEO_COMPRESSION_BITRATE = 600000;
+  private static final int HIGH_VIDEO_COMPRESSION_HEIGHT  = 480;
+  private static final int HIGH_VIDEO_COMPRESSION_WIDTH  = 640;
+  private static final int HIGH_VIDEO_COMPRESSION_BITRATE =  450000;
```

Comment:
Maybe shove these in a map-like structure instead so adding new ones is easy?

Reply:
I'm not a biggest fan of having it his way, but in all the classes that are children of this one, they have lists of constants like this, so i wanted to proceed with similar practices.